nerdlets/tag-improver-nerdlet/index.js

```diff
@@ -331,21 +297,33 @@ export default class TagVisualizer extends React.Component {
   };
 
   updatePolicy = policy => {
-    this.setState({
-      taggingPolicy: sortedPolicy(policy),
-      mandatoryTagCount:
-        policy.filter(tag => tag.enforcement === 'required').length || 0
-    });
+    this.setState(
+      {
+        taggingPolicy: sortedPolicy(policy),
+        mandatoryTagCount:
+          policy.filter(tag => tag.enforcement === 'required').length || 0
+      },
+      () => {
+        const { tagHierarchy, mandatoryTagCount } = this.state;
+        const { updateEntityTagCompliance } = this;
+        Object.entries(tagHierarchy).forEach(tagKey => {
```

Comment:
Not a blocker but I've started using `for (tagKey of Object.entries(tagHierarchy)) { ... }` instead of `.forEach(tagKey => {...})`. Do you have a preference or opinion on which one is better for simple iterative processing like this?

Reply:
no preference on that. 
I can change `"forEach"` loops in `"index.js"` to `for (... of Object.entries(...)) {...}`